usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -31,6 +45,60 @@ def universal_transaction_matview_annotations():
         "usaspending_permalink": Concat(
             Value(AWARD_URL), Func(F("transaction__award__generated_unique_award_id"), function="urlencode"), Value("/")
         ),
+        "disaster_emergency_fund_codes_for_overall_award": Case(
+            When(
+                action_date__gte="2020-04-01",
+                then=Subquery(
+                    FinancialAccountsByAwards.objects.filter(
+                        award_id=OuterRef("award_id"), disaster_emergency_fund__isnull=False
+                    )
+                    .annotate(
+                        value=ExpressionWrapper(
+                            Concat(
+                                F("disaster_emergency_fund__code"),
+                                Value(": "),
+                                F("disaster_emergency_fund__public_law"),
+                            ),
+                            output_field=TextField(),
+                        )
+                    )
+                    .values("award_id")
+                    .annotate(total=StringAgg("value", ";", distinct=True))
+                    .values("total"),
+                    output_field=TextField(),
+                ),
+            )
+        ),
+        "outlayed_amount_funded_by_COVID-19_supplementals_for_overall_award": Case(
+            When(
+                action_date__gte="2020-04-01",
+                then=Subquery(
+                    FinancialAccountsByAwards.objects.filter(
+                        filter_by_latest_closed_periods(),
+                        award_id=OuterRef("award_id"),
+                        disaster_emergency_fund__group_name="covid_19",
+                    )
+                    .values("award_id")
+                    .annotate(sum=Sum("gross_outlay_amount_by_award_cpe"))
+                    .values("sum"),
+                    output_field=DecimalField(),
+                ),
+            ),
+        ),
+        "obligated_amount_funded_by_COVID-19_supplementals_for_overall_award": Case(
+            When(
+                action_date__gte="2020-04-01",
+                then=Subquery(
+                    FinancialAccountsByAwards.objects.filter(
+                        award_id=OuterRef("award_id"), disaster_emergency_fund__group_name="covid_19"
```

Comment:
we don't filter_by_latest_closed_periods here?

Reply:
obligations and outlays are summed differently, outlays are basically balances so we only want to sum the values from the last entry in a closed period. obligations are transactional so we sum all of them.